.gitignore

```diff
@@ -37,6 +37,9 @@ rerun.txt
 coverage.data
 coverage/*
 
+## RUBYMINE
```

Comment:
Please move this to your own global `.gitignore`. rather than pushing this into our repository. https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Reply:
Will do. Thanks for the link!

Reply:
✅